### PR TITLE
Read data from Terabee TOF sensor

### PR DESCRIPTION
**Pull request by Yuanbwcx**
## Main Changes

`Evo_single_point_display_range_py3.py` can be used to detect and read data from Terabee TOF sensor

## Tests

This has been tested and works well at 15m and 60m one.

### CODE REVIEW

File: raspberry/bzzz/sensors/TeraBee_Evo_time_of_flight_sensor/Evo_single_point_display_range_py3.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    ports = list(serial.tools.list_ports.comports())
+    for p in ports:
+        # print p # This causes each port's information to be printed out.
+        if "5740" in p[2]:
+            print('Evo found on port ' + p[0])
+            return p[0]
+    return 'NULL'
+
+
+def openEvo(portname):
```
Comment (alphaville) [inline raspberry/bzzz/sensors/TeraBee_Evo_time_of_flight_sensor/Evo_single_point_display_range_py3.py:26]: Function names should be snake-case (see [here](https://github.com/QUB-ASL/bzzz/blob/main/CONTRIBUTING.md#32-python))

File: raspberry/bzzz/sensors/TeraBee_Evo_time_of_flight_sensor/Evo_single_point_display_range_py3.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```
Comment (alphaville) [inline raspberry/bzzz/sensors/TeraBee_Evo_time_of_flight_sensor/Evo_single_point_display_range_py3.py:1]: This code, taken from the sample codes of Evo, is in the form of a script. For us to be able to use it in this project we need to make it into a class. I would suggest the following structure:

```python
class EvoSensor:

    def __init__(self, port, baud=115200):
       # initialise Evo sensor

    def read(self):
        # returns reading from sensor
```

**Review by alphaville — CHANGES_REQUESTED** [review]
@jamie-54 can you help Siyuan with this?

**Comment by jamie-54** [comment]
> @jamie-54 can you help Siyuan with this?

yea

**Comment by alphaville** [comment]
@Yuanbwcx @jamie-54 How is it going guys? Has this stagnated?

**Comment by alphaville** [comment]
In this PR we should probably do the save as in #137.

**Comment by alphaville** [comment]
@jamie-54 Why was this PR closed?

**Pull request closed without merging**